src/MFRC522v2.h

```diff
@@ -136,7 +136,7 @@ class MFRC522 {
   /////////////////////////////////////////////////////////////////////////////////////
   // Functions for manipulating the MFRC522
   /////////////////////////////////////////////////////////////////////////////////////
-  bool PCD_Init();
+  bool PCD_Init(bool antenna_on = true);
```

Comment:
What is the use case for this?
Why not a simple  `PCD_AntennaOff();` after `PCD_Init()`?